### PR TITLE
migrate to next13

### PR DESCRIPTION
**Pull request by rafinskipg**
Upgraded to nextjs and did some manual testing through the pages, seems to work ok. 

No added new features from nextjs

### CODE REVIEW

File: modules/app/stores/uiFilters.ts [hunk 1]
```diff
@@ -6,7 +6,7 @@ SPDX-License-Identifier: AGPL-3.0-or-later
 
 */
 
-import create from 'zustand';
+import { create } from 'zustand';
```
Comment (adamgoth) [inline modules/app/stores/uiFilters.ts:9]: Just wondering why this was necessary. Doesn't seem zustand version was updated.

**Review by adamgoth — APPROVED** [review]
We should do team-wide QA on develop before we go to prod because it's so many changes, but did a quick walkthrough the deployed URL and things looked good. Site seems quicker 💨

File: modules/app/stores/uiFilters.ts (hunk 1 above)
Comment (rafinskipg) [reply modules/app/stores/uiFilters.ts:9]: When running the portal it kept complaining that is deprecated and that we should import it like this now

**Review by torhector2 — APPROVED** [review]
💯

**Pull request merged**